x-pack/plugin/sql/qa/src/main/resources/geo/geosql.csv-spec

```diff
@@ -214,3 +214,55 @@ SELECT ST_Distance(ST_WktToSql(NULL), ST_WktToSQL('POINT (-71 42)')) shape;
   shape:d
 null
 ;
+
+groupByGeometryType
+SELECT COUNT(*) cnt, ST_GeometryType(location) gt FROM geo GROUP BY ST_GeometryType(location);
+
+     cnt:l     |      gt:s
+15             |POINT
+;
+
+groupByEastWest
+SELECT COUNT(*) cnt, FLOOR(ST_X(location)/90) east FROM geo GROUP BY east ORDER BY east;
+
+     cnt:l     |     east:l
+3              |-2
+3              |-1
+4              |0
+5              |1
+;
+
+groupByNorthSouth
+SELECT COUNT(*) cnt, FLOOR(ST_Y(location)/45) north FROM geo GROUP BY north ORDER BY north;
```

Comment:
Can you add another test where you GROUP BY both `north` and `east` and then ORDER BY `east` and `north` (notice the change in ordering compared to group by ordering)?